src/engine/RuntimeInformation.cpp

```diff
@@ -66,18 +66,20 @@ void RuntimeInformation::writeToStream(std::ostream& out, size_t indent) const {
       << '\n';
   out << indentStr(indent) << "columns: " << absl::StrJoin(columnNames_, ", ")
       << '\n';
-  out << indentStr(indent) << "total_time: " << totalTime_ << " ms" << '\n';
-  out << indentStr(indent) << "operation_time: " << getOperationTime() << " ms"
+  out << indentStr(indent) << "total_time: " << totalTime_.count() << " ms"
       << '\n';
+  out << indentStr(indent) << "operation_time: " << getOperationTime().count()
+      << " ms" << '\n';
   out << indentStr(indent) << "status: " << toString(status_) << '\n';
   out << indentStr(indent)
       << "cache_status: " << ad_utility::toString(cacheStatus_) << '\n';
   if (cacheStatus_ != ad_utility::CacheStatus::computed) {
-    out << indentStr(indent) << "original_total_time: " << originalTotalTime_
-        << " ms" << '\n';
     out << indentStr(indent)
-        << "original_operation_time: " << originalOperationTime_ << " ms"
+        << "original_total_time: " << originalTotalTime_.count() << " ms"
```

Comment:
Add a comment "`TODO<g++12, Clang 17>` use `<< originalTotalTime_` directly".
(It is part of C++20, but not all of our current c ompilers support it.